packages/ripple-test-tools/step_definitions/common/components.js

```diff
@@ -74,6 +89,47 @@ Then(`the campaign primary title should be {string}`, title => {
   cy.get('.rpl-campaign-primary__title').should('contain', title)
 })
 
+Then(`the campaign primary banner should contain:`, (dataTable) => {
+  const primaryCampaign = dataTable.rawTable.slice(1)
+  primaryCampaign.forEach((link, index) => {
+    cy.get('.rpl-campaign-primary').eq(index).then(campaign => {
+      const title = link[0].trim()
+      const body = link[1]
+      const ctatext = link[2]
+      const ctalink = link[3]
+      expect(campaign).to.contain.text(title)
+      expect(campaign).to.contain.text(body)
+      cy.wrap(campaign).get('.rpl-campaign-primary__call-to-action').then(ctalinkobj => {
+        expect(ctalinkobj).to.contain.text(ctatext)
+        expect(ctalinkobj).to.contain.attr('href', ctalink)
+      })
+    })
+  })
+})
+
+// Campaign secondary
+Then(`the campaign secondary component should exist`, () => {
+  cy.get('.rpl-campaign-secondary').should('exist')
+})
+
+Then(`the campaign secondary component should contain:`, (dataTable) => {
+  const primarySecondary = dataTable.rawTable.slice(1)
```

Comment:
@alan-cole If you think a helper function for working with dataTables would be useful we can add a utilities library or something. It would just mean importing into the step definition file as required. I think this is perfectly fine for now.